dash-pipeline/tests/saithrift/ptf/vnet/test_saithrift_vnet.py

```diff
@@ -90,13 +90,12 @@ def setUp(self):
 
     def runTest(self):
         # TODO form a packet related to dataplane config
+        # Verify pipeline drop - Form a pkt to invalid VIP
         self.udp_pkt = simple_udp_packet()
-        # TODO expected packet might be different
-        self.udp_pkt_exp = self.udp_pkt
-        print("\nSending packet...", self.udp_pkt.__repr__())
+        print("\nSending packet to invalid vip...", self.udp_pkt.__repr__())
         send_packet(self, 0, self.udp_pkt)
-        print("\nVerifying packet...", self.udp_pkt_exp.__repr__())
-        verify_packet(self, self.udp_pkt_exp, 0)
+        print("\nVerifying packet drop...")
+        verify_no_other_packets(self, 0)
```

Comment:
Note in some environments this might fail because of IPv67 control-plane packets etc. bombarding the veths. So if this starts failing in people's dev environments that might be why.